### ISSUE TITLE
private session: special charaters in username stripped (works in regular session)

### ISSUE THREAD

**Issue by datenbank-projekt** [udima-university/moodle-mod_jitsi#74]
Hi,

I notice, that a name with (german) special characters like ö,ä,ü and ß is not shown correctly in private sessions: The letter is stripped.
What wonders me, is that the letters are correctly ported to the jisty session when it is not a private session, but regular activity.

Olaf

**Comment by datenbank-projekt** [comment]
Hi,

also, in private sessions the name is set to lower case - which it is not in "regular" sessions.
Olaf

**Comment by SergioComeron** [comment]
This is related to the last modification that fixed an xss problem. I think all the private session functionality needs to be modified and the feature will be retired in the next version before being released again.

**Comment by datenbank-projekt** [comment]
If you mean the private sessions will be retired I am crying for that ;-) I think it's a great feature to enable the self learning person to communicate and cooperate with others.

**Comment by SergioComeron** [comment]
No, no. I have big plans for private sessions. But I think that now they are not being well implemented. I will work these days to see if I can rethink them solving these problems. But if not, I think the best thing is to remove it now and redo it correctly.
